Products/Zuul/routers/device.py

```diff
@@ -363,6 +363,14 @@ def _getInfoData(self, info, keys):
         return values
 
     @require('Manage Device')
+    def resumeCollection(self, id):
+        uid = id.split('/')[-1]
+        facade = self._getFacade()
+        device = facade._getObject(id)
+        device.resumeCollection(uid)
```

Comment:
Logic should be moved into the facade. By moving there will also only be one object load instead of two;  replaces the object load and then the search.